File: app/src/main/java/com/example/mobileappdevelopment/Coordinates.java
package com.example.mobileappdevelopment;

import com.google.android.gms.maps.model.LatLng;

import java.util.ArrayList;
import java.util.List;

class Coordinates {

    private static List<LatLng> coordinates = new ArrayList<>();

    static void addCoordinates(LatLng latLng) {
        coordinates.add(latLng);
    }

    static List<LatLng> getCoordinatesList() {
        return coordinates;
    }
}

File: app/src/main/java/com/example/mobileappdevelopment/utils/Coordinates.kt
package com.example.mobileappdevelopment.utils

import com.google.android.gms.maps.model.LatLng
import kotlin.collections.ArrayList

object Coordinates {
    var coordinates: MutableList<LatLng> = ArrayList()
    fun addCoordinates(latLng: LatLng?) {
        this.coordinates.add(latLng!!)
    }

//    val coordinates: MutableList<LatLng>
//        get() = this.coordinates
}